src/types/drive.ts

```diff
@@ -204,14 +206,14 @@ export interface DriveCurrentFolderContent {
   folderContent: DriveItemData[];
 }
 
-export type FolderContentChild = Omit<FolderChild, 'uuid'>;
+export type FolderContentChild = FolderChild;
 
 export type FolderContent = Omit<FetchFolderContentResponse, 'children'> & {
   children: FolderContentChild[];
 };
 
 export interface FetchFolderContentResponseWithThumbnails extends FolderContent {
-  files: (DriveFileData & { thumbnail?: DownloadedThumbnail })[];
+  files: (DriveFileData & { thumbnail?: DownloadedThumbnail; uuid?: string })[];
```

Comment:
betted to add uuid to DriveFileData, we will need this in more places

Reply:
`DriveFileData` is a type from sdk, we need to add it to the sdk and adapt all apps types.

Reply:
use an alias then, let's try to decouple the Internxt SDK types and the Mobile types, as they don't need to match, we will refactor this on AFS mobile